presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestElasticsearch7IntegrationSmokeTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.elasticsearch;
+
+import static java.lang.String.format;
+
+public class TestElasticsearch7IntegrationSmokeTest
+        extends TestElasticsearchIntegrationSmokeTest
+{
+    TestElasticsearch7IntegrationSmokeTest()
+    {
+        super("7.0.0");
+    }
+
+    @Override
+    protected String indexEndpoint(String index, String docId)
+    {
+        return format("/%s/_doc/%s", index, docId);
```

Comment:
Why are we assigning a docId to documents? this was previously unnecessary?

Reply:
high level api generates a random id..
all documents have an _id field, this is that as in low level api we have to provided it.. is generated automatic but we have to put it on url